codespell_lib/data/dictionary.txt

```diff
@@ -1997,6 +1999,7 @@ braodcasted->broadcasted
 Brasillian->Brazilian
 brazeer->brassiere
 bre->be
+breaked->broken
```

Comment:
break in past tense should be changed to `broke` not 'broken'

Reply:
Fixed. Removed the commit, and added the good one.